cmd/agent/daemon/conntrack/conntrack_cilium_linux.go

```diff
@@ -0,0 +1,170 @@
+//go:build linux
+
+package conntrack
+
+import (
+	"net/netip"
+	"path/filepath"
+
+	"github.com/castai/kvisor/pkg/logging"
+	"github.com/castai/kvisor/pkg/proc"
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/defaults"
+	"github.com/cilium/cilium/pkg/loadbalancer"
+	"github.com/cilium/cilium/pkg/maps/ctmap"
+	"github.com/cilium/cilium/pkg/maps/lbmap"
+	"github.com/cilium/cilium/pkg/tuple"
+	"github.com/cilium/cilium/pkg/u8proto"
+)
+
+var (
+	ciliumCt4    *bpf.Map
+	ciliumCt6    *bpf.Map
+	backends4Map *bpf.Map
+	backends6Map *bpf.Map
+)
+
+func iniCiliumMaps(log *logging.Logger) bool {
+	var err error
+
+	ciliumCt4, err = bpf.OpenMap(proc.HostPath(filepath.Join(defaults.BPFFSRoot, defaults.TCGlobalsPath, ctmap.MapNameTCP4Global)), &ctmap.CtKey4Global{}, &ctmap.CtEntry{})
```

Comment:
This looks a bit dangerous. Would this work if the version of cilium we bundle and the one running on the host be different and there are changes to the structs?

Reply:
ct maps should be stable, from that I saw they try to keep it compatible. For some other maps they have versioning like `Backend4ValueV3` 
But do we even have any option here?